feature-libs/product-configurator/rulebased/components/attribute/show-options/configurator-show-options.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container>
+  <button
```

Comment:
We have to talk to our accessibility expert whether `Show Options` buttons is read properly

Reply:
I think we miss a reference to the attribute name at this point.

Reply:
@steinsebastian: maybe we need a aria text describing what the button does?